app/public/js/localSprites.js

```diff
@@ -176,6 +209,7 @@ function addHomingFireballSprite(xPosition, yPosition, target) {
         $.extend(rectangle(4 * 32, 0 * 32, 32, 32), {image: fireballBImage, hitBox}),
     ];
     var homingFireballSprite = new SimpleSprite({frames}, xPosition, yPosition, 0, 0, 1.5, 1.5);
+    homingFireballSprite.name = 'homingFireball';
```

Comment:
Instead of name, let's use `type` here and use the consts I suggested above so this could be:
`homingFireballSprite.type = OBJECT_TYPE_FIREBALL`
And actually, since these are all "sprites" maybe we should name these consts like `SPRITE_TYPE_FIREBALL` and we can add `SPRITE_TYPE_TRIGGER` as well. So I guess it would be:
`homingFireballSprite.type = SPRITE_TYPE_FIREBALL`